lib/stencil-init.js

```diff
@@ -37,15 +37,30 @@ class StencilInit {
      * @returns {Promise<void>}
      */
     async run (dotStencilFilePath, cliOptions = {}) {
+
+        const cliConfig = this.getCliConfig(cliOptions);
         const oldStencilConfig = this.readStencilConfig(dotStencilFilePath);
-        const defaultAnswers = this.getDefaultAnswers(oldStencilConfig, cliOptions);
-        const answers = await this.askQuestions(defaultAnswers);
-        const updatedStencilConfig = this.applyAnswers(oldStencilConfig, answers);
+        const defaultAnswers = this.getDefaultAnswers(oldStencilConfig);
+        const questions = this.getQuestions(defaultAnswers, cliConfig);
+        const answers = questions ? await this.askQuestions(questions) : {};
```

Comment:
Let's move this ternary operator inside `askQuestions`. It will be easier to test it there. And the `run` method will just contain the integration code with no logic

Reply:
kinda:
```
async askQuestions (questions) {
    return questions.length ? this.inquirer.prompt(questions) : {};
}
```